drivers/wifi/nrf700x/zephyr/inc/zephyr_wpa_supp_if.h

```diff
@@ -74,5 +77,13 @@ void wifi_nrf_wpa_supp_event_proc_deauth(void *if_priv,
 void wifi_nrf_wpa_supp_event_proc_disassoc(void *if_priv,
 					   struct img_umac_event_mlme *disassoc,
 					   unsigned int event_len);
+
+void wifi_nrf_wpa_supp_event_proc_get_station(void *if_priv,
+					   struct img_umac_event_new_station *info,
+					   unsigned int event_len);
+
+void wifi_nrf_wpa_supp_event_proc_get_interface(void *if_priv,
```

Comment:
You can use "if" instead of "interface" in function name.

Reply:
Done.